Added Inline latex rendering documentation (#5257)
mattermost/docs: source/messaging/formatting-text.rst

Math Formulas
-------------

.. tabs::

  .. tab:: Using Inline LaTeX

>     From Mattermost v6.1, you can create formulas that display inline using LaTeX. Begin and end each formula with a dollar sign ($) symbol.

Reviewer: ```suggestion
    From Mattermost v6.1, you can create formulas that display inline using LaTeX. Use the dollar sign ($) symbol at the beginning and end of each formula.
```

Reviewer: This may be easier to read?
